jit-compiler/src/codegen.rs

```diff
@@ -150,9 +150,7 @@ impl<'a, T: FieldElement> CodeGenerator<'a, T> {
                     }
                 })
             }
-            _ => Err(format!(
-                "Definition of this kind not supported: {symbol} - {definition}"
```

Comment:
Removed this because this might call the wrong display function and lead to a panic.